CONTRIBUTING.md

```diff
@@ -74,37 +75,34 @@ export class AppModule {
 }
 ```
 - if you are using `ng-bootstrap` as peer dependency, you have to add `import '@angular/localize/init'` in your `app/polyfill.ts`
-- build your application in prod mod with `ng build --prod --output-hashing=none search-wc` at the root of the project. 
 
-To use your webcomponent in a website, you can
-- merge all the build output javascript files into one and GZIP it
-```shell script
-cat dist/search-wc/{runtime-es2015,polyfills-es2015,main-es2015}.js | gzip > gn-search-snapshot.js.gz
-```
-- include it in your HTML file
-```angular2html
-...
-    <script src="gn-search-snapshot.js"></script>
-...
-    <gn-search-snapshot></gn-search-snapshot>
-```
-and run your html file in a web server
+#### Test your Web Component
+1. Storybook
 
-You can for example 
--Add a http-server library
-```
-npm install http-server
+You can visualise your web component as it would be in a standalone web page with Storybook.
+```shell script
+npm run storybook-wc
 ```
-add a new scripts in your `package.json`
+2. Full deployment
 
+To test your web component in true a web page, run
 ```shell script
- "serve-wc": "http-server --gzip",
+npm run serve-wc -- (application_name) (webcomponent_tag_name) [--build]
```

Comment:
I think the logic here is a bit misleading... I had trouble understanding what was the difference between `application_name` and `webcomponent_tag_name`. It's working for now but maybe we should have a list of webcomponents somewhere with their respective angular app and tag name, and read from that instead?

Reply:
I have refactored the whole thing to make web component applications simpler, also angular application name and WC tag name should be the same